### ISSUE TITLE
Web UI 0.6.0-rc1 gives a blank page when external services are registered

### ISSUE THREAD

**Issue by bastichelaar** [hashicorp/consul#1406]
The web UI gives a blank page when an external service is registered in Consul. Steps to reproduce:

- Install Consul 0.6.0-rc1
- Run the external service example:
```
$ curl -X PUT -d '{"Datacenter": "dc1", "Node": "google",
   "Address": "www.google.com",
   "Service": {"Service": "search", "Port": 80}}'
   http://127.0.0.1:8500/v1/catalog/register
```
- Try to open the web UI

The page will remain blank. When you remove the service, the UI is available again. Javascript console error:

```javascript
Uncaught TypeError: Cannot read property 'reduce' of null
```

**Comment by slackpad** [comment]
Thanks for the report! We will take a look.

**Comment by slackpad** [comment]
Confirmed this was fixed by #1414.
